aiida/manage/manager.py

```diff
@@ -123,8 +124,10 @@ def get_backend_manager(self):
         :return: the database backend manager
         :rtype: :class:`aiida.backend.manager.BackendManager`
         """
+        from aiida.backends import get_backend_manager
```

Comment:
not quite following the logic here - in your other change you were placing the import inside the if clause...

Reply:
place of the import wasn't done intentional, could have been in either place and is not really that important.